api/openapi/users.yml

```diff
@@ -1967,4 +1967,3 @@ components:
 security:
   - bearerAuth: []
   - refreshAuth: []
-  
```

Comment:
Add new line

Reply:
It already has a new line
![image](https://github.com/mainflux/mainflux/assets/100555904/3d31cca8-f1a0-4522-bc1e-85b4165ba024)

I'm not sure why its showing otherwise in github

Reply:
Probably because it is not correctly formatted

Reply:
The .yml file? I have checked with swagger UI it has no issues. Or what formatting were you referring to?

Reply:
It is not about Swagger UI, but the general convention and consistency with other files.